test/specs/mounting-options/localVue.spec.js

```diff
@@ -17,14 +17,14 @@ describeWithMountingMethods('options.localVue', mountingMethod => {
         template: `
         <div>{{test}}</div>
       `,
-        data: { test: '' }
+        data: () => ({ test: '' })
       }
       const localVue = Vue.extend()
       localVue.version = '2.3'
       const wrapper = mountingMethod(TestComponent, {
-        localVue: localVue,
-        mocks: { test: 'some value' }
+        localVue: localVue
       })
+      wrapper.vm.test = 'some value'
```

Comment:
@eddyerburgh I had to remove the `mocks: { test: 'some value'}` and replace it with this, otherwise the test woudl fail.

I *suspect* that this is because you can't actually mock properties for proxied data properties, but that didn't fail before because the `data: { test: '' }` itself failed?

Reply:
This test is the first one written for localVue, and itcould be rewritten. The intent is to check that VTU mounts using the localVue constructor, so a better test would create a localVue constructor with createLocalVue, add a property to the localVue prototype, and mount a component that relies on that property existing.

Would you be able to rewrite it?

Reply:
Will try tomorrow

Reply:
might take a few days until I really have the time :/